CIPs/cip-0032.md

```diff
@@ -27,11 +27,12 @@ Despite Validator uptime on Celo being phenomenal, the maintenance of the AS nod
 
 The `Validators.distributeEpochPaymentsFromSigner()` method will be updated to take into account the status and performance of the signer's Attestation Service. The `maxPayment` will be divided into two payments, with 80% going to the operation of the Validator and 20% going to the operation of the Attestation Service. The 80% will act as the new base reward for the validator (`actual reward = base * score * slashingMultiplier`).
 
-The Attestation Reward will be affected directly by the completion rate of the Attestation Service during each epoch. Since a 100% completion rate is unrealistic, a governable target completion rate will be used to determine the rewards. To ensure that an Attestation Service isn't incorrectly penalized by a bug outside of their responsibilities (ex. Valora), a minimum global attestation completion rate for this epoch will be first considered. If the minimum global attestation completion rate is not met, all elected Attestation Services will be rewarded with 75% the maximum amount, regardless of their score. A registered Attestation Service can be forcibly deregistered if they don't meet the minimum number of attestations over a governable sliding window. The Attestation Reward will be forfeited if the validator does not have a registered Attestation Service.
+The Attestation Reward will be affected directly by the completion rate of the Attestation Service during each epoch. Since a 100% completion rate is unrealistic, a governable target completion rate will be used to determine the rewards. To ensure that an Attestation Service isn't incorrectly penalized by a bug outside of their responsibilities (ex. Valora), a minimum global attestation completion rate for this epoch will be first considered. The minimum global attestation completion rate is a dynamic value calculated by applying the `MinGlobalAttestationCompletionRate` to the moving average from the last `AttestationBaselineEpochCount` epochs (ex. global completion rate shouldn't dip below .3 * avg completion rate of the last 30 epochs). If the minimum global attestation completion rate is not met, all elected Attestation Services will be rewarded with 75% the maximum amount, regardless of their score. A registered Attestation Service can be forcibly deregistered if they don't meet the minimum number of attestations over a governable sliding window. The Attestation Reward will be forfeited if the validator does not have a registered Attestation Service.
 
 - `TargetCompletionRate` - The target completion rate for Attestation Services, taking into account standard user drop-off and other factors outside of the AS control.
 - `AttestationServiceRewardPercentage` - The percentage of max rewards that are allocated to running an Attestation Service. Initially `.20`.
```

Comment:
Just a callout that based on feedback we have gotten on contract release 3, we should aim to have the smart contract changes deployed with parameters that do not change the system and then pass another governance proposal to set the new parameters. In this case, we would want to set this to 0, and make a proposal that lands after the release to set it to `0.20`

Reply:
Good to know. Do you have any pointer to the discussion? I'm curious why that is.

Reply:
https://forum.celo.org/t/core-contracts-release-3/755/2?u=victor

Reply:
Got it, this is more about governance and community acceptance than any technical limitation. I totally agree that we should break apart CGPs wherever possible